### PR TITLE
KD-9787 | Update email-service for campaigns-service

### PR DESCRIPTION
**Pull request by aiden-kudos**
# What Does this PR do?
This PR will support extra field for campaign service
https://kudoswork.atlassian.net/browse/KD-9787

The reason why we are going to create v1.1.1 is that I don't want to impact other services because we don't know exactly where this one is being used. The v1.1.1 will be used by campagins-service.

# How do I test this?
- Create release v1.1.1
- Use this release in Campaign-service

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ const postToSengrid = ({
     subject,
     reply_to: from,
     template_id: templateId,
+    categories: [category],
+    custom_args: JSON.stringify(uniqueArgs),
```
Comment (aiden-kudos) [inline src/index.js:27]: Depending on the version of the API or how it is sent email, this field has a different name.

File: src/index.js [hunk 2]
```diff
@@ -48,8 +50,13 @@ const paramsHtml = (params) => {
 };
 
 const postToSmtp = ({
-  from, to, subject, params, html,
+  from, to, subject, params, html, category, uniqueArgs,
 }) => {
+  const smtpApiHeader = {
+    category: [category],
+    unique_args: uniqueArgs,
```
Comment (aiden-kudos) [inline src/index.js:57]: Depending on the version of the API or how it is sent email, this field has a different name.

File: src/index.js (hunk 1 above)
Comment (aiden-kudos) [reply src/index.js:27]: https://docs.sendgrid.com/for-developers/sending-email/unique-arguments

File: src/index.js (hunk 2 above)
Comment (aiden-kudos) [reply src/index.js:57]: https://docs.sendgrid.com/for-developers/sending-email/unique-arguments

**Review by abideen-kudos — APPROVED** [review]

**Pull request merged**